client/network/src/service.rs

```diff
@@ -828,12 +828,8 @@ impl<B: BlockT + 'static, H: ExHashT> NetworkService<B, H> {
 		request: Vec<u8>
 	) -> Result<Vec<u8>, RequestFailure> {
```

Comment:
```suggestion
		request: Vec<u8>,
		connect: IfDisconnectedBehaviour
	) -> Result<Vec<u8>, RequestFailure> {
```

Assuming that the only difference between `request` and `send_request`/other-name would be the channel, I'd suggest to add `IfDisconnectedBehaviour` here as well.

Reply:
This would be a breaking change though, I guess I should this would then be release notes worthy?

Reply:
This is done with a `B5` label